src/opera/test/pge/dist_s1/test_dist_s1_pge.py

```diff
@@ -100,6 +102,8 @@ def generate_band_data_output(self, product_id, band_data, empty_file=False, cle
             Product ID to create the output files for.
         band_data: tuple of str
             Files to add to the output directory.
+        directory: str
+            Directory to create the test files in.
```

Comment:
This should note that this is an optional argument, and what the default is if not provided.